code/datums/components/crafting/recipes.dm

```diff
@@ -927,15 +927,15 @@
 	result = /obj/item/stack/medical/bruise_pack/one
 	time = 1
 	reqs = list(/obj/item/stack/medical/gauze = 1,
-				/datum/reagent/medicine/styptic_powder = 10)
+				/datum/reagent/medicine/styptic_powder = 40)
 	category = CAT_MISC
 
 /datum/crafting_recipe/burn_pack
 	name = "Burn Ointment"
 	result = /obj/item/stack/medical/ointment/one
 	time = 1
 	reqs = list(/obj/item/stack/medical/gauze = 1,
-				/datum/reagent/medicine/silver_sulfadiazine = 10)
+				/datum/reagent/medicine/silver_sulfadiazine = 40)
```

Comment:
Undocumented changes

Reply:
Also 40 is an insane amount for 1 ointment and makes them even less worth making

Reply:
It's to make it equal with brute/burn patches. 1u of styptic is equal to one brute damage. 1u of silver sulfadaizine is equal to one burn damage. They're very much still worth making, no stamina penalty for applying them and you can stack them.